cumulusci/tasks/bulkdata/step.py

```diff
@@ -445,6 +452,12 @@ def __init__(self, *, sobject, operation, api_options, context, fields):
             for field in getattr(context.sf, sobject).describe()["fields"]
         }
         self.boolean_fields = [f for f in fields if describe[f]["type"] == "boolean"]
+        self.api_options["batch_size"] = (
```

Comment:
@prescod should this also .copy() the api_options the way you did in BulkApiDmlOperation?